### DISCUSSION TITLE
Invalid json response body for npx create-next-app --ts (FETCH_ERROR)

### DISCUSSION THREAD

**Discussion by ahamedfazil** [vercel/next.js#28449] [Help]
### What version of Next.js are you using?

latest (11.1.0)

### What version of Node.js are you using?

10.24.1

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10

### How are you deploying your application?

next export

### Describe the Bug

I am trying to create new nextjs project with Typescript support and I run the below command 

`npx create-next-app --ts` 

following this doc - https://nextjs.org/docs/basic-features/typescript

but it throws an error,

![image](https://user-images.githubusercontent.com/21005625/130436082-b8398188-3a29-4376-9775-913cc4484b34.png)

### Expected Behavior

New nextjs app should be created with typescript support.

### To Reproduce

run - `npx create-next-app --ts`

**Comment by G3root** [comment]
try running
`npm cache clean --force`

**Comment by ahamedfazil** [comment]
still i get same error after cleaning cache. The command `npx create-next-app --ts` was working fine few weeks ago but now not

![image](https://user-images.githubusercontent.com/21005625/130473459-51f5c78e-7be5-441f-bc09-3a66a151ec75.png)

**Comment by ahamedfazil** [comment]
Looks like there is issue with my node. Its not only happening for create-next-app but it happens for other package. I uninstalled my node and did fresh install. Also set my proxy to null by following this link resolved my issue -> https://github.com/nodejs/help/issues/2569#issuecomment-702089435
